lua/dap/repl.lua

```diff
@@ -9,6 +9,8 @@ local history = {
   idx = 1
 }
 
+local autoscroll = vim.fn.has('nvim-0.7') == 1
```

Comment:
Hey @mfussenegger, could this support nvim 0.7 and above? So it would support nvim 0.8 as well?

Reply:
`vim.fn.has('nvim-0.7') == 1` is true on nvim 0.8 or later as well